src/components/FeeSelector/shared.tsx

```diff
@@ -0,0 +1,30 @@
+import { Trans } from '@lingui/macro'
+import { FeeAmount } from '@uniswap/v3-sdk'
+import { SupportedChainId } from 'constants/chains'
+import { ReactNode } from 'react'
+
+export const FEE_AMOUNT_DETAIL: Record<
+  FeeAmount,
+  { label: string; description: ReactNode; supportedChains: SupportedChainId[] | 'all' }
+> = {
+  [FeeAmount.LOWEST]: {
```

Comment:
Would it be better to label them after bits?

Reply:
probably 😓 that would have to be done in the v3-sdk though :/

Reply:
Ah in that case probably not worth th refactor